ironic/conf/drac.py

```diff
@@ -21,7 +21,14 @@
                min=1,
                help=_('Interval (in seconds) between periodic RAID job status '
                       'checks to determine whether the asynchronous RAID '
-                      'configuration was successfully finished or not.'))
+                      'configuration was successfully finished or not.')),
+    cfg.IntOpt('query_management_config_job_status_interval',
+               default=120,
+               min=1,
+               help=_('Interval (in seconds) between periodic MANAGEMENT job '
```

Comment:
Could you change MANAGEMENT to lower case on the above line and below?

Reply:
Sure Chris.